tests/unit/s2n_connection_size_test.c

```diff
@@ -45,7 +45,7 @@ int main(int argc, char **argv)
     }
 
     /* Carefully consider any increases to this number. */
-    const uint16_t max_connection_size = 4274;
+    const uint16_t max_connection_size = 4288;
```

Comment:
Call-out for reviewers: this test only failed on the BSD CI job, and I'm not sure why. I suspect it is somehow related to adding the `bool available` field to `s2n_kem_group`, which should add an additional byte to the struct. However, from what I can tell, `s2n_kex_parameters` (of which `s2n_connection` has 1) only has pointers to `s2n_kem_group` structs (or pointers to arrays thereof). I'm not sure where/how 14 (with 1 byte added each) concrete `s2n_kem_group` structs would come into the mix, or why it would only manifest on BSD.

Reply:
This shouldn't be necessary anymore, right?

Reply:
right, we shouldn't. will update.